google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1alpha2/BQTableSchemaToProtoDescriptorTest.java

```diff
@@ -183,7 +184,7 @@ public void testStructComplex() throws Exception {
             .build();
     final Table.TableFieldSchema TEST_NUMERIC =
         Table.TableFieldSchema.newBuilder()
-            .setType(Table.TableFieldSchema.Type.NUMERIC)
```

Comment:
Why change this to BYTES? It is called TEST_NUMERIC.

Reply:
I changed it here just to avoid having to also change the v1alpha2 JstonToProtoMessage and other supporting changes. This test references the proto/jsonTest.proto file that I changed in order to test the changes made in v1beta2. This just sets the type here to byte since we changed the numeric to be also be bytes. This just makes the test pass so that maven doesn't get angry

Reply:
nvm